### PR TITLE
Clearier environment instructions

### PR DESCRIPTION
**Pull request by pinussilvestrus**
Closes https://github.com/bpmn-io/form-js/issues/214

This
* adds a note about our development environment requirements
* replaces `cp` with `copyfiles` npm package that should work regardless

### CODE REVIEW

**Comment by pinussilvestrus** [comment]
Consider: adding a note about supported node version.

**Comment by pinussilvestrus** [comment]
@christian-konrad @Skaiir I updated the PR to be more transparent on our supported Node dev environments and added a link to our contribution guidelines (the main one from bpmn-io). Feel free to challenge that, e.g., if you think we need even more guidance.

I thought about adding information right at the top to jump to the contribution guidelines + build instructions quickly. As I don't see such a prominent banner somewhere that often, I wouldn't do that.

File: README.md [hunk 1]
```diff
@@ -69,11 +69,16 @@ See [playground documentation](./packages/form-js-playground) for further detail
...
 * [Changelog](./packages/form-js/CHANGELOG.md)
+* [Contributing guide](https://github.com/bpmn-io/.github/blob/master/.github/CONTRIBUTING.md#create-a-pull-request)
 * [Form schema](./docs/FORM_SCHEMA.md)
 
 
 ## Build and run
 
+Build the project in a Posix environment. On Windows, that is [Git Bash](https://gitforwindows.org/) or WSL. 
+
+Note we currently support development environments with Node.js version 16 (and npm version 8). We encourage you to use a Node.js version manager (e.g., [`nvm`](https://github.com/nvm-sh/nvm) or [`n`](https://github.com/tj/n)) to set up the needed versions.
```
Comment (vsgoulart) [inline README.md:80]: Shouldn't we add a `.nvmrc` to the project then?

Comment (pinussilvestrus) [reply README.md:80]: Good catch; I just added it via 90c536d 👍

**Review by vsgoulart — APPROVED** [review]

**Pull request merged**